src/components/Dropdown/DropdownMessageNoResults.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react'
+
+import { ReactProps } from '../../../types/utils'
+import {
+  commonPropTypes,
+  ContentComponentProps,
+  createShorthandFactory,
+  UIComponent,
+  UIComponentProps,
+} from '../../lib'
+import ListItem from '../List/ListItem'
+
+export interface DropdownMessageNoResultsProps
+  extends UIComponentProps<DropdownMessageNoResultsProps>,
+    ContentComponentProps {}
+
+/**
+ * A DropdownMessageNoResults is a sub-component that outputs a message when a list of filtered
+ * items is empty.
+ */
+class DropdownMessageNoResults extends UIComponent<ReactProps<DropdownMessageNoResultsProps>> {
+  static className = 'ui-dropdown__message-no-results'
+  static create: Function
+  static displayName = 'DropdownMessageNoResults'
+  static propTypes = commonPropTypes.createCommon({ children: false })
+
+  renderComponent({ classes, ElementType, unhandledProps }) {
+    const { content } = this.props
+
+    return <ListItem className={classes.root} content={content} {...unhandledProps} />
```

Comment:
Not sure about this new components. Can't we use the Box component with `as={ListItem}`? We can still create slots in the styles for them... Thoughts?

Reply:
I decided to reuse just `ListItem` there 👍